spec/controllers/v1/actions_controller_spec.rb

```diff
@@ -1,8 +1,39 @@
 require 'rails_helper'
 
 RSpec.describe V1::ActionsController, type: :controller do
+  render_views
+
+  describe "GET count" do
+    before(:all) do
+      @main_activity = FactoryGirl.create(:activity)
+      FactoryGirl.create_list(:action, 3, activity: @main_activity)
+      Timecop.freeze(4.days.ago) do
+        FactoryGirl.create_list(:action, 2)
+      end
+    end
+    it "gets all actions count" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.